internal/internal_worker.go

```diff
@@ -1209,7 +1224,234 @@ func (aw *aggregatedWorker) Stop() {
 	aw.logger.Info("Stopped Worker")
 }
 
-// aggregatedWorker returns an instance to manage both activity and decision workers
+// WorkflowReplayer is used to replay workflow code from an event history
+type WorkflowReplayer struct {
```

Comment:
nit: Is it possible to move the entire `WorkflowReplayer` component to a new file?

Reply:
Yes. I will move a couple of stuffs to their own files. Don't want to do it in this diff.